server/users/users.controller.js

```diff
@@ -12,8 +12,12 @@ router.route("/").post(bodyParser.json(), async (request, response) => {
     user.createdAt = Date.now();
     await user.save();
     return response.status(200).json("User created!");
-  } catch (error) {
-    return response.status(400).send(error);
+  } catch (e) {
+    console.log(e);
```

Comment:
somewhere in your next PR, try remove this console log.. ill still approve it this time tho 🥇